projects/plugins/jetpack/modules/plugin-search.php

```diff
@@ -358,14 +358,7 @@ public function inject_jetpack_module_suggestion( $result, $action, $args ) {
 
 			// Try to match a passed search term with module's search terms.
 			foreach ( $jetpack_modules_list as $module_slug => $module_opts ) {
-				/*
-				* Does the site's current plan support the feature?
-				* We don't use Jetpack_Plan::supports() here because
-				* that check always returns Akismet as supported,
-				* since Akismet has a free version.
```

Comment:
D75985-code also stops reporting `akismet` as available on free sites, so I took the opportunity to simplify this code.

Reply:
Just realized this won't work until https://github.com/Automattic/wp-calypso/issues/61542 is sorted out first, so I'm reverting this changes.